CHANGELOG.md

```diff
@@ -1,5 +1,34 @@
 # Change history for ui-invoice
 
+## [1.2.0](https://github.com/folio-org/ui-invoice/tree/v1.2.0) (2019-12-04)
+[Full Changelog](https://github.com/folio-org/ui-invoice/compare/v1.1.0...v1.2.0)
+
+### Stories
+* [UINV-66](https://issues.folio.org/browse/UINV-66) View invoice adjustments accordion
+* Common payment methods usage
```

Comment:
it's a code\technical debt, I'm not sure we need to specify this here without appropriate ticket. Could you please leave this aside? thanks